blackbox/docs/admin/system-information.rst

```diff
@@ -1826,6 +1826,71 @@ node.
 
    The ``sys.allocations`` table is subject to :ref:`shard_table_permissions`.
 
+.. _sys-segments:
+
+Segments
```

Comment:
I think it makes sense to add this section directly after the `sys.shards` documentation section.